stuffer/s2n_stuffer_network_order.c

```diff
@@ -167,29 +167,31 @@ static int length_matches_value_check(uint32_t value, uint8_t length)
     return S2N_SUCCESS;
 }
 
-static int s2n_stuffer_write_reservation_impl(struct s2n_stuffer_reservation reservation, uint32_t u)
+static int s2n_stuffer_write_reservation_impl(struct s2n_stuffer_reservation* reservation, uint32_t u)
```

Comment:
Should it be `const`?

Reply:
`u` yes, `reservation` no. I'll update it.